src/UI/Component/Input/Container/Factory.php

```diff
@@ -29,4 +29,71 @@
+	 *   effect: >
+	 *      In the Filter Bar:
+	 *      Clicking on the Expand/Collapse Glyph expands/collapses the second area of the Filter,
+	 *      where the Filter Fields are placed. When the Filter Bar is expanded, the row with set filter settings
+	 *      at the bottom of the Filter Bar will be hidden.
+	 *      Clicking on the "Apply" Bulky Button applies the settings which the user has made for the Filter and
+	 *      reloads the page immediately.
+	 *      Clicking on the "Reset" Bulky Button resets the Filter to the initial state and reloads the page immediately.
+	 *      Clicking on the "Toggle" Button imitates a click on the "Apply" Bulky Button, activates/deactivates the Filter
+	 *      and reloads the page immediately.
+	 *      In the Filter Fields Area:
+	 *      Clicking on a Filter Field between its Label and its "Remove" Glyph shows up a Popover where an Input Field
+	 *      is presented. Writing down a value into the Input Field synchronizes the value in the Input Field
+	 *      with the value in the Filter Field.
+	 *      Clicking on the "Remove" Glyph in a Filter Field makes a Filter Field disappear from the Filter.
+	 *      Clicking on the "Add" Bulky Button shows up a Popover with a list of all possible Filter Fields which are not
+	 *      part of the Filter yet. Clicking on one specific Filter Field in this list adds the selected Filter Field
+	 *      to the Filter and imitates a click on it.
+	 *   rivals:
+	 *      forms: >
+	 *          Unlike Filters, Forms are used to enter or modify data in the system.
+	 *
+	 * rules:
+	 *   usage:
+	 *     1: Filters MUST be used on the same page as tables or other collections of items.
+	 *   interaction:
+	 *     1: Filter Fields MUST NOT be editable, just clickable.
+	 *   wording:
+	 *     1: Labels of Filter Fields MUST be shown shortened (with three dots at the end) when space is scarce.
+	 *     2: The set filter settings in the bottom row of the Filter Bar MUST be shown shortened when space is scarce.
+	 *   style:
+	 *     1: The Filter Bar and the Filter Fields Area SHOULD be separated visually, e.g. with a border-line.
+	 *     2: The Toggle Button MUST NOT contain a Label.
+	 *     3: The Popovers SHOULD be shown below the Filter Field or the "Add" Bulky Button.
+	 *   ordering:
+	 *     1: A Filter MUST be placed above the item collection it acts upon.
+	 *   responsiveness:
+	 *     1: >
+	 *        On screens larger than medium size, there MUST be three Filter Fields per row. On medium-sized screens
+	 *        or below, only one Filter Field MUST be shown per row.
+	 *     2: >
+	 *        On screens larger than small size, the "Apply" and "Reset" Bulky Buttons MUST contain a glyph plus text.
+	 *        On small-sized screens, these Bulky Buttons MUST contain only a glyph.
+	 *   accessibility:
+	 *     1: Filter Fields MUST be accessible by keyboard by using the "Tab"-Key and clickable by using the "Return"-Key.
```

Comment:
This is what we would like to achieve, too. Not sure how complicated this will be in the implementation, since the wrapped Input Fields will have internal tab handling, too. This might become a tricky part.